feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
I get your point about loading the unresolved answers on click. Considering your own example - will any user go ahead and click on each of his/her 50 explorations to see their stats ? No. In most cases he/she might click on a few or even not click on any of them at all! (After all, it's important to consider that every type of user, and not just creators are redirected to the dashboard when they open https://oppia.org, assuming they are logged in). So it makes sense to load the bare minimum data for rendering the dashboard page and keeping it as fast as possible!
^I do believe that it would've been better if this realization would've come to me earlier 😛 

Also, that's a whole lot of queries in the above example (200 * 10 * 50 = 100000 unique unresolved answers!) which leads to an even bigger amount of data being transferred.